client/app/src/Service/RequestIdLoggerProcessor.php

```diff
@@ -32,30 +33,53 @@ public function __construct(ContainerInterface $container)
     }
 
     /**
-     * Add request header 'x-request-id' into ['extra']['request_id']
+     * Add request header 'x-aws-request-id' into ['extra']['aws_request_id']
      * Does not change the record if the scope is not active, or the request is not found or doesn't contain the header.
      *
      * @return array same record with extra info
      */
     public function processRecord(array $record)
     {
         $reqId = self::getRequestIdFromContainer($this->container);
+        $sessId = self::getSessionSafeIdFromContainer($this->container);
 
         if ($reqId) {
-            $record['extra']['request_id'] = $reqId;
+            $record['extra']['aws_request_id'] = $reqId;
+        }
+
+        if ($sessId) {
+            $record['extra']['session_safe_id'] = $sessId;
         }
 
         return $record;
     }
 
-    public static function getRequestIdFromContainer(ContainerInterface $container)
+    public static function getRequestIdFromContainer(ContainerInterface $container): ?string
```

Comment:
Same comments as for the API version.

Ideally I'd like to avoid this code being in two places, but we don't have a shared components package used by client and api. We really ought to.

Reply:
It's not for now, but the other area where we have duplication is in the entities. We should probably share those across the frontend and the API as they are practically the same.